README_WINDOWS.md

```diff
@@ -1,16 +1,16 @@
 #  bitpit on Microsoft Windows
 
-This is a short guide to set a bitpit compliant 64bit Windows environment using MSYS2/MingGW64.
+This is a short guide to set a bitpit compliant 64bit Windows environment using MSYS2/MinGW64.
 
 ## Requirements:
 - <B>Windows 8/10</B> (8.1 pro/10 tested)
 
-- <B>MSYS2</B> (64bit) https://www.msys2.org/
+- <B>MSYS2</B> (64bit) https://www.msys2.org/ (current version msys2-x86_64-20210419.exe)
```

Comment:
Instead of "current version" can we say something like "tested version"/"recommended version"? Otherwise this guide will become outdated as soon as a new version of MinGW is released.

Reply:
No objection really, it's fine for me. Even if it's really hard to choose the  version of msys2/mingw. Usually they expose the latest version and stop. Periodically we need to update and retest the installation chain (every 6months at least or if major stuff happened msys2 side)